json-schemas/interface-methods/events-subscribe.json

```diff
@@ -0,0 +1,47 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "https://identity.foundation/dwn/json-schemas/events-subscribe.json",
+    "type": "object",
+    "additionalProperties": false,
+    "required": [
+      "descriptor"
```

Comment:
Sanity: `authorization` is required also no?

Reply:
Yeah it should be.

I think I initially looked at `records-query.json` as a scaffold. It also doesn't exist there, is that on purpose?
https://github.com/TBD54566975/dwn-sdk-js/blob/b3a75778bcae8bbe0acc4168e8655d09a5415477/json-schemas/interface-methods/records-query.json#L6

Reply:
Yes, that's on purpose for `RecordsQuery`, since it can be an anonymous query.